### ISSUE TITLE
Pause buttons are shown only once

### ISSUE THREAD

**Issue by YariKartoshe4ka** [YariKartoshe4ka/Space-Way#13]
When the pause is called for the first time, the buttons are shown as expected, but with subsequent ones, they disappear and are no longer shown

https://user-images.githubusercontent.com/49284924/141651320-704cba94-7c2e-421b-a034-b26713e9dee9.mov

**Comment by YariKartoshe4ka** [comment]
The problem is that the buttons change their Y coordinate to **infinite**, so the button just isn't within the window
```
$ python "Space Way.py"

Subscene changed to `pause`!
<ResumeButton Sprite(in 2 groups)> <ellipse(353, 193.5, 63, 63)>
Subscene changed to `game`!
Subscene changed to `pause`!
<ResumeButton Sprite(in 2 groups)> <ellipse(353, inf, 63, 63)>
```

**Comment by YariKartoshe4ka** [comment]
Fixed with [6ae4fdb](https://github.com/YariKartoshe4ka/Space-Way/commit/6ae4fdbdcbfe8a22edeb6f6d89685b4164138b74)
